server/sonar-server/src/main/java/org/sonar/server/user/ws/HomepageTypes.java

```diff
@@ -26,9 +26,9 @@
   enum Type {
     PROJECT,
     /**
-     * This type in only available on SonarQube
+     * These types in only available on SonarQube
```

Comment:
typo: These types are available only on Sonarqube

Reply:
I saw an important improvement in the quality of your PR, congratulations for that!

Reply:
thx, takes time ...